user-service/controller/user-controller.js

```diff
@@ -24,7 +25,7 @@ export async function createUser(req, res) {
                 return res.status(201).json({message: `Created new user ${username} successfully!`});
             }
         } else {
-            return res.status(400).json({message: 'Username and/or Password are missing!'});
+            return res.status(400).json({message: 'Username/Email/Password are missing!'});

```

Comment:
```suggestion
            return res.status(400).json({ message: 'Username/Email/Password is/are missing!' });

```

Reply:
Since you're editing this file already, for all `message`s applicable, do you mind adding a space after `{` and before `}` for consistency sake and change `!` to `.` as it seems abit harsh when prompting the users haha.